csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -434,60 +434,52 @@ void HopperMultipleMatmulScheduler::scheduleMmaResults() {
   }
 }
 
-void HopperMultipleMatmulScheduler::scheduleOutputTensor(TensorView* c) {
-  // Block Schedule and Parallelize
-  blockTileTensors({c});
-  parallelizeBlocks({c});
-
-  // Apply mma common transformation
-  c->split(-2, getM(params_->mma_macro));
-  c->split(-1, getN(params_->mma_macro));
-  // [..., Mo, No, Mio, Mii, Nio, Nii]
-  // -> [..., Mo, No, Mio, Nio, Mii, Nii]
-  c->reorder({{-3, -2}});
-  c->merge(-4);
-
-  // [...., Mii, Nii] ->
-  // [...,  Mii/16,  Miioi(2), Miii(8), Nii/8, Niio(4), Niii(2)] ->
-  // [.., Mii/16,  Miii(8), Niio(4),  Nii/8, Miioi(2), Niii(2)   ]
-  // [..., Mii/16 * Miii(8) * Niio(4), Nii/8, Miioi(2), Niii(2)  ]
-  // Mii/16 * Miii(8) * Niio(4) is 128 for Hopper and this is parallelized as
-  // TIDx.
-  auto s =
-      mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(c->getLoopDomain());
-  c->setLoopDomain(s.as<IterDomain*>());
-  c->axis(-5)->parallelize(ParallelType::TIDy);
-}
-
 void HopperMultipleMatmulScheduler::scheduleEpilogue() {
+  // Load/cache the epilogue inputs if there are any.
+  auto& c_tvs = tensor_roles_.at(MatmulTensorRole::EPILOGUE_INPUT);
+  std::vector<TensorView*> cached_tvs;
+  for (auto* c : c_tvs) {
+    cached_tvs.push_back(c->cacheAfter());
+  }
+
+  // Propato to (not including) the splitk output if there is a splitk
+  // else this is just mma_results_
```

Comment:
I don't think this is true. We only fill out `splitk_sums_` if `splitk_factor != 1`.